parser/src/lexer/mod.rs

```diff
@@ -102,6 +102,12 @@ pub enum Token {
     #[token(".")]
     Dot,
 
+    #[token("(")]
+    Lpar,
+
+    #[token(")")]
+    Rpar,
+
```

Comment:
nit: for some reason, I find this strange to read, and if I didn't see the token next to it I'm not sure that I'd immediately understand what it means. Maybe it would be clearer just do `Lparen`, since it's not that long and "paren" is already an abbreviation (so it's still consistent with our use of abbreviations)